CHANGELOG.md

```diff
@@ -437,4 +439,4 @@ final transaction is created by calling `finish` on the builder.
 [v0.16.0]: https://github.com/bitcoindevkit/bdk/compare/v0.15.0...v0.16.0
 [v0.16.1]: https://github.com/bitcoindevkit/bdk/compare/v0.16.0...v0.16.1
 [v0.17.0]: https://github.com/bitcoindevkit/bdk/compare/v0.16.1...v0.17.0
-[unreleased]: https://github.com/bitcoindevkit/bdk/compare/v0.17.0...HEAD
\ No newline at end of file
+[unreleased]: https://github.com/bitcoindevkit/bdk/compare/v0.17.0...HEAD
```

Comment:
I am seeing an extra blank line beneath this.. But its not showing in the change set in github.. So have a double check..

Reply:
good catch. it looks like it's caused by I edited this line using vim which add newline at the end of each line. I have to remove the last line and use `echo -n` to avoid making change 😅